frontend/views/utils/vError.js

```diff
@@ -29,9 +29,9 @@ Vue.directive('error', {
     if (!vnode.context.$v.form[binding.arg]) {
       throw new Error(`v-error: vuelidate doesn't have validation for ${binding.arg}`)
     }
-    const opts = binding.value
+    const opts = binding.value || {}
     const pErr = document.createElement(opts.tag || 'span')
-    for (const attr in (opts.attrs || {})) {
+    for (const attr in (opts.attrs)) {
```

Comment:
Style: in this case, just `opt.attrs` instead of `(opts.attrs)`, since there is no longer a `|| {}` in there